xarray/core/dataarray.py

```diff
@@ -1637,7 +1643,8 @@ def combine_first(self, other):
         """
         return ops.fillna(self, other, join="outer")
 
-    def reduce(self, func, dim=None, axis=None, keep_attrs=None, **kwargs):
+    def reduce(self, func, dim=None, axis=None, keep_attrs=None,
```

Comment:
```suggestion
    def reduce(self, func, dim=None, axis=None, keep_attrs=False,
```

Reply:
This seems outside the scope of the current change

Reply:
Ah, wrong one. Thanks